### PR TITLE
feat: add rate limit and call policy

### PR DESCRIPTION
**Pull request by adnpark**

### CODE REVIEW

File: packages/test/v0.7/permissionValidator.test.ts [hunk 1]
```diff
@@ -61,235 +122,197 @@ describe("Permission kernel Account", () => {
...
+            expect(response).toMatch(TX_HASH_REGEX)
+            console.log("Transaction hash:", response)
+        },
+        TEST_TIMEOUT
+    )
+
+    test(
+        "Smart account client send transaction with SignaturePolicy",
+        async () => {
+            const signaturePolicy = await toSignaturePolicy({
```
Comment (derekchiang) [inline packages/test/v0.7/permissionValidator.test.ts]: let's rename this to "signature caller" policy.  So `toSignatureCallerPolicy`.

File: packages/test/v0.7/permissionValidator.test.ts [hunk 2]
```diff
@@ -61,235 +122,197 @@ describe("Permission kernel Account", () => {
...
-            //         preVerificationGas: 84700n,
-            //         callGasLimit: 1273781n,
-            //         verificationGasLimit: 726789n
-            //     }
-            // })
-            const response = await kernelClient.sendTransaction({
-                to: zeroAddress,
-                value: 0n,
-                data: "0x"
+            const rateLimitPolicy = await toRateLimitPolicy({
```
Comment (derekchiang) [inline packages/test/v0.7/permissionValidator.test.ts:215]: I suggest adding a more complex use case that leverages all the flags on this policy.

**Review by derekchiang — CHANGES_REQUESTED** [review]

File: plugins/permission/policies/toSudoPolicy.ts
Comment (SahilVasava) [inline plugins/permission/policies/toSudoPolicy.ts:1]: why was this deleted?

Comment (adnpark) [reply plugins/permission/policies/toSudoPolicy.ts:1]: ah I asked Taek, and he said that the sudo policy is no longer needed, so I removed it

Comment (SahilVasava) [reply plugins/permission/policies/toSudoPolicy.ts:1]: Ah! I see. Gonna check.

**Pull request merged**